app/src/main/java/space/taran/arknavigator/ui/fragments/utils/Preview.kt

```diff
@@ -8,17 +9,31 @@ enum class PredefinedIcon {
     FOLDER, FILE
 }
 
-data class Preview(val predefined: PredefinedIcon? = null, val image: Path? = null) {
+data class Preview(
+    val predefinedFolder: PredefinedIcon? = null,
```

Comment:
Maybe `predefinedIcon`?

Reply:
> Maybe `predefinedIcon`?

Initially, "predefined" was used to determine if the file has a file or folder icon. With current implementation, this variable serves only one purpose - to determine whether the icon has a folder icon, or a variation of the file icon. Perhaps, in the future we will have some differentiations of the folder icon, hence the "PredefinedIcon" class. The name itself "predefined" was very confusing when writing if-else statements for icon sources, so I renamed it. I will turn it back.

Reply:
`PredefinedIcon` has 2 constructors: `FOLDER` and `FILE`.
`FILE` isn't used anymore, so just `val isFolder: Boolean` should be enough.
We don't need this enum anymore.